### PR TITLE
Added new tablet banner property

### PR DESCRIPTION
**Pull request by Erislandio**
What is the purpose of this pull request?
Banner property new action for tablet devices

What problem is this solving?
banner resolution on tablet devices

How should this be manually tested?
go to https://www.melimeloparis.ro/

na home leaves browser resolution at 1024px

note that the banner loses image quality as it is mobile.

### CODE REVIEW

**Comment by klzns** [comment]
There are still requested changes to be done. Let me know if they need to be clarified.

**Comment by Erislandio** [comment]
@klzns Its done bro, pls review it <3

File: messages/en.json [hunk 1]
```diff
@@ -20,5 +20,8 @@
   "admin/editor.carousel.banner.description.title": "Banner description",
   "admin/editor.carousel.banner.externalRoute.title": "External route",
   "admin/editor.carousel.banner.image.title": "Banner image",
-  "admin/editor.carousel.banner.mobileImage.title": "Banner mobile image"
+  "admin/editor.carousel.banner.mobileImage.title": "Banner mobile image",
+  "admin/editor.carousel.banner.tabletImage.title": "Bbanner tablet image"
```
Comment (klzns) [inline messages/en.json]:
```suggestion
  "admin/editor.carousel.banner.tabletImage.title": "Banner tablet image"
```

File: messages/en.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
...
+
+
```
Comment (klzns) [inline messages/en.json]: Remove those empty lines

**Comment by Erislandio** [comment]
@klzns Can you review it again pls?

**Comment by klzns** [comment]
I'll publish tomorrow morning. Thanks for the PR!

**Pull request merged**